app/components/post_textbox/post_textbox.js

```diff
@@ -111,6 +117,7 @@ export default class PostTextbox extends React.PureComponent {
                         ref='input'
                         value={this.props.value}
                         onChangeText={this.handleTextChange}
+                        onSelectionChange={this.autocomplete ? this.autocomplete.handleSelectionChange : () => true}
```

Comment:
Can you make this line into a method? I'm thinking about enabling the [`jsx-no-bind`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md) rule since we've done so well with avoiding creating unnecessary functions when rendering.